feature/google/src/latestDependencies/java/com/revenuecat/purchases/google/billingClientParamBuilders.kt

```diff
@@ -41,3 +47,27 @@ val SubscriptionOfferDetails.subscriptionBillingPeriod: String?
 
 val SubscriptionOfferDetails.isBasePlan: Boolean
     get() = this.pricingPhases.pricingPhaseList.size == 1
+
+fun PurchaseOption.buildPurchaseParams(storeProduct: StoreProduct): BillingFlowParams? {
+    val googlePurchaseOption = this as? GooglePurchaseOption
+    if (googlePurchaseOption == null) {
+        errorLog("PurchaseOption for a Play purchase must be a GooglePurchaseOption.") //TODOBC5: Improve and move error message
```

Comment:
Agreed about moving the error message up

Reply:
yeah i might as well do that in this PR... will update